src/k8s/components/indexed-search/generate/statefulset/statefulset.dhall

```diff
@@ -43,20 +46,29 @@ let Fixtures/global = ../../../../util/test-fixtures/package.dhall
 
 let tc = Fixtures/statefulset.indexed-search.Config
 
+let Util/ListToOptional = ../../../../util/functions/list-to-optional.dhall
+
 let PodSpec/generate
     : ∀(c : Configuration/Internal/StatefulSet) → Kubernetes/PodSpec.Type
     = λ(c : Configuration/Internal/StatefulSet) →
-        Kubernetes/PodSpec::{
-        , containers =
-              [ Container/zoekt-webserver/generate c.Containers.zoekt-webserver
-              , Container/zoekt-indexserver/generate
-                  c.Containers.zoekt-indexserver
-              ]
-            # c.sideCars
-        , securityContext = Some Kubernetes/PodSecurityContext::{
-          , runAsUser = Some 0
-          }
-        }
```

Comment:
Ran `volumes` function through utility function to have it be some iff there was a volume present. Another question. I hard coded the "data" volume here because it was missing originally. Was that intentional?

Reply:
hm, not intentional. data volume is actually important, it's where the index shards are

Reply:
yeah, i definitely must have missed these things.